app/src/main/java/com/example/pocketcloset/SaveClothingView.java

```diff
@@ -186,7 +186,7 @@ public void done(ParseException e) {
 
             }
         });
-        Log.i("SaveClothingView", "Clothing:" + article.getClothingType().toString());
+        Log.i("SaveClothingView", "Clothing:" + article.getClothingType());
```

Comment:
fwiw, using `%s` is really what you want.
`Log.i("SaveClothingView", "Clothing: %s", article.getClothingType())`